src/components/common/applicant-info-1.vue

```diff
@@ -1,133 +1,190 @@
                 :messages="messages['middleName']"
                 :value="applicant.middleName"
                 :rules="firstMiddleNameRules"
-                @blur="handleNameBlur('middleName', $event)"
-                @input="updateApplicant('middleName', $event)"
                 dense
                 filled
                 height="50"
                 hide-details="auto"
-                id="middlename"
                 :name="Math.random()"
                 autocomplete="chrome-off"
                 label="Middle Name (Optional)"
+                @blur="handleNameBlur('middleName', $event)"
+                @input="updateApplicant('middleName', $event)"
               />
             </v-col>
 
-            <v-col cols="12" md="4" lg="4" class="pt-0">
-              <label for="lastname" class="hidden">Last Name</label>
+            <v-col
+              cols="12"
+              md="4"
+              lg="4"
+              class="pt-0"
+            >
+              <label
+                for="lastname"
+                class="hidden"
+              >Last Name</label>
               <v-text-field
+                id="lastname"
                 :messages="messages['lastName']"
                 :rules="lastNameRules"
                 :value="applicant.lastName"
-                @blur="handleNameBlur('lastName', $event)"
-                @input="updateApplicant('lastName', $event)"
                 dense
                 filled
                 height="50"
                 hide-details="auto"
-                id="lastname"
                 :name="Math.random()"
                 autocomplete="chrome-off"
                 label="Last Name"
+                @blur="handleNameBlur('lastName', $event)"
+                @input="updateApplicant('lastName', $event)"
               />
             </v-col>
           </v-row>
 
           <!--ADDDRESS MENU / STREET ADDRESS-->
           <v-row class="mt-2 mb-0">
-            <v-col cols="12" class="py-0 my-0">
+            <v-col
+              cols="12"
+              class="py-0 my-0"
+            >
               <v-menu
+                v-model="showAddressMenu"
                 bottom
                 class="pa-2"
                 dense
                 max-width="600"
                 offset-y
-                v-model="showAddressMenu"
               >
-                <!-- Using activator slot to avoid having to absoluetly position v-menu. Visibility is controlled by
-                "showAddressMenu" field, "on" function provided by slot is used only to supress error message-->
-                <template v-slot:activator="{ on }">
+                <!-- Using activator slot to avoid having to absolutely position v-menu. Visibility is controlled by
+                "showAddressMenu" field, "on" function provided by slot is used only to supress error message. -->
+                <!-- eslint-disable-next-line vue/no-unused-vars -->
+                <template #activator="{ on }">
```

Comment:
"on" is not used here (weird workaround per comment above) so I simply ignored the eslint problem here.

I will test this before merging.

Reply:
Works fine.

![image](https://github.com/bcgov/namerequest/assets/10002889/51ffbd41-ee54-4e46-aa79-16739be36947)